pkg/scheduler/framework/plugins/tainttoleration/taint_toleration.go

```diff
@@ -98,7 +98,7 @@ func getAllTolerationPreferNoSchedule(tolerations []v1.Toleration) (tolerationLi
 // PreScore builds and writes cycle state used by Score and NormalizeScore.
 func (pl *TaintToleration) PreScore(ctx context.Context, cycleState *framework.CycleState, pod *v1.Pod, nodes []*v1.Node) *framework.Status {
 	if len(nodes) == 0 {
-		return nil
+		return framework.NewStatus(framework.Skip)
```

Comment:
score will not run anyways if there are no nodes to score, so this PR is not necessary.